### DISCUSSION TITLE
Public URL code

### DISCUSSION THREAD

**Discussion by vincehi** [riccardoperra/codeimage#626] [Ideas]
Hello, great tool.
I would really like to be able to share the public URL to the snippet in order to copy and paste the code for people who have seen the screenshot on social networks with a great link.
What do you think ?

**Comment by riccardoperra** [comment]
Hi, currently as a logged in user you can actually share the link to everyone

https://app.codeimage.dev/0baed200-a5eb-4148-9f44-40a2214c66b3

Interface can be still improved
